### DISCUSSION TITLE
Running mww3_case03 and 4 cased in regtests are failed?

### DISCUSSION THREAD

**Discussion by lj-cug** [NOAA-EMC/WW3#755] [General]
After conduct ./w3_make, I try to run the regtests and the three cases downloaded from FTP. Most of the cases are run successfully, but 4 cases in regtests and the third case from FTP runnings are failed. I give the screen output, could you please give me some suggestion about where is the problem.
![mww3_case03_error](https://user-images.githubusercontent.com/53997605/183322954-01e36cd9-5b75-49d0-866c-3011080dbde7.png)
![regtest_case1_error](https://user-images.githubusercontent.com/53997605/183322970-d43355b5-a2cd-4d73-9fc0-320255f72fc1.png)

Thanks
Li Jian
China University of Geosciences

**Comment by aliabdolali** [comment]
@lj-cug You can run regtests using regtests/bin/run_cmake_test which does the prep, main job and post jobs in order.
Alternatively, you can compile the code with the switch in the input directory, but need to be careful how to execute and rename binary files. 
For example, in order to execute ww3_ounf, you need out_grd.ww3 and corresponding mod_def.ww3 in your work directory.

**Comment by lj-cug** [comment]
Dear Sir:
   My another problem is run the third case downloaded from FTP. The running go wrong, could you please give me some hints? Thanks!
![mww3_case03_error](https://user-images.githubusercontent.com/53997605/183331292-9b1edb02-7be7-4009-8e83-71d132c56fda.png)

**Comment by lj-cug** [comment]
Dear Sir:
    About the problem running regtest, I conduct "./bin/run_test -n 16 -p mpirun -s PR3_UQ_MPI ../model mww3_test_01 "
   I cannot find out_grd.ww3 and mod_def.ww3 in the work directory, so could you please tell me how to generate these two files? I am curious that most of the case runnings have no problem.
Thanks
